airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/__init__.py

```diff
@@ -141,6 +152,7 @@
 
 RESOURCES = {
     **METADATA_RESOURCE_TREE,
+    **CONNECTOR_METRICS_RESOURCE_TREE,
```

Comment:
💅 I'm pretty aware about which process feeds the listed resources, but not this new one. Shall we document in comments or in the orchestrator readme the data sources we use to have an high level lineage about what gets into the registry and from where.